custom/icds_reports/management/commands/stale_data_in_ucr.py

```diff
@@ -95,7 +95,7 @@ def _get_stale_data(run_config):
         for chunk in chunked(matching_records_for_db, chunk_size):
             doc_ids = [val[0] for val in chunk]
             ucr_insertion_dates = _get_ucr_insertion_dates(run_config.domain, run_config.table_id, doc_ids)
-            for doc_id, doc_type, sql_modified_on in chunk:
+            for doc_id, doc_type, sql_modified_on, _ in chunk:
```

Comment:
where did this come from?

Reply:
`matching_records_for_db` here comes from `_get_primary_data_for_db` which in turn calls `get_sql_case_data_for_db`, which is a function imported from `stale_data_in_es` which I changed in the same commit (https://github.com/dimagi/commcare-hq/pull/26961/commits/23b5a0e28a0b352fbc96dcf811a242518c279855). I noticed though I probably need to change the other conditional fork in `_get_primary_data_for_db` as well or better yet push this "drop the domain" behavior into `_get_primary_data_for_db` itself.

Thanks for asking, looks like in the process of answering I found a bug :)

Reply:
@czue what do you think about dropping the section where we look up a case's case_type if it's not in the input file? https://github.com/dimagi/commcare-hq/blob/7923460997afedb252dfb4a35e2267665df5acff/corehq/apps/hqadmin/management/commands/republish_doc_changes.py#L116-L131. Looks like previously this command was made for dealing with just a list of IDs, and that's where this behavior comes from, but now if you're following the process the input file will already have this info. Also, other functionality has since been added, and it's likely that other things break the ability to import with just a list of ids. Thoughts?